tests/CollectionDataProviderTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+use Nayjest\Grids\CollectionDataProvider;
+use Nayjest\Grids\EloquentDataRow;
+use Nayjest\Grids\Grids;
+use Illuminate\Pagination\Paginator;
+use Illuminate\Database\Eloquent\Collection;
+
+class CollectionDataProviderTest extends TestCase
+{
+
+
+    /**
+     * Creates the fake collectiondataprovider for testing
+     *
+     * @return Nayjest\Grids\CollectionDataProvider
+     */
+    private function getTestCollectionDataProvider()
+    {
+        $models = factory(App\User::class, 3)->make();
+        return new CollectionDataProvider($models);
+    }
+
+    /**
+     * Create a Collection Data Provider
+     *
+     * @return \Nayjest\Grids\CollectionDataProvider
+     */
+    public function test_create_CollectionDataProvider()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method test_create_CollectionDataProvider is not named in camelCase.](https://www.codacy.com/app/mail_2/Grids/file/1994350266/issues/source?bid=1990429&fileBranchId=3120437#l29)